### ISSUE TITLE
assumes a global 'berror' installation

### ISSUE THREAD

**Issue by jaredly** [knowbody/crra#31]
should probably have error handling & just ignore if not there

**Comment by nikgraf** [comment]
Having the same issue

```
Hang tight! Starting your app...events.js:160
      throw er; // Unhandled 'error' event
      ^

Error: spawn berror ENOENT
    at exports._errnoException (util.js:1022:11)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:193:32)
    at onErrorNT (internal/child_process.js:359:16)
    at _combinedTickCallback (internal/process/next_tick.js:74:11)
    at process._tickCallback (internal/process/next_tick.js:98:9)
error Command failed with exit code 1.
```

**Comment by keirah** [comment]
This looks like the same issue as this one - https://github.com/knowbody/crra/issues/29. I'll close that one, so that we don't have multiple open.

**Comment by knowbody** [comment]
this should be fixed now by #35
